### PR TITLE
Nifi-4918 JMS Connection Factory setting the dynamic Properties wrong

### PR DESCRIPTION
**Pull request by jugi92**
Thank you for submitting a contribution to Apache NiFi.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? Is it referenced 
     in the commit message?

- [x] Does your PR title start with NIFI-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.

- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [x] Have you ensured that the full suite of tests is executed via mvn -Pcontrib-check clean install at the root nifi folder?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file under nifi-assembly?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file found under nifi-assembly?
- [ ] If adding new Properties, have you added .displayName in addition to .name (programmatic access) for each of the new properties?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/cf/JMSConnectionFactoryProvider.java [hunk 1]
```diff
@@ -257,21 +256,26 @@ private void setConnectionFactoryProperties(ConfigurationContext context) {
      */
     private void setProperty(String propertyName, Object propertyValue) {
         String methodName = this.toMethodName(propertyName);
-        Method method = Utils.findMethod(methodName, this.connectionFactory.getClass());
-        if (method != null) {
+        Method[] methods = Utils.findMethods(methodName, this.connectionFactory.getClass());
+        if (methods != null && methods.length < 0) {
```
Comment (markap14) [inline nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/cf/JMSConnectionFactoryProvider.java]: methods.length can never be less than 0 here... was that supposed to be > 0 perhaps?

Comment (jugi92) [reply nifi-nar-bundles/nifi-jms-bundle/nifi-jms-processors/src/main/java/org/apache/nifi/jms/cf/JMSConnectionFactoryProvider.java]: Yes sorry, it was supposed to be methods.length > 0
I will change and open a new pull request

**Comment by jugi92** [comment]
@markap14  could you check again and merge if everything is as expected?

**Comment by markap14** [comment]
@jugi92 thanks for the update, code looks correct to me now. Was able to verify that things still are working the way that I expect. Thanks! Merged to master.

**Pull request closed without merging**